pywbem/cim_obj.py

```diff
@@ -778,20 +783,17 @@ def _cim_keybinding(key, value):
         return value
 
     # pylint: disable=unidiomatic-typecheck
-    if builtin_type(value) in six.integer_types + (float,):
-        # Ideally, we won't accept keybinding values specified as Python int or
-        # float values, but require a CIM data type (e.g. Uint32 or Real32).
-        # However, that was allowed in earlier versions of pywbem, so for
-        # compatibility, we deprecate that but do not raise an exception.
+    if builtin_type(value) in number_types:
         # Note: The CIM data types are derived from the built-in types, so we
         # cannot use isinstance() for this test.
-        msg = "Setting a CIMInstanceName keybinding to a %s value is " \
-            "deprecated; use a CIM data type (e.g. Uint32) instead." % \
-            builtin_type(value)
-        if DEBUG_WARNING_ORIGIN:
-            msg += "\nTraceback:\n" + ''.join(traceback.format_stack())
-        warnings.warn(msg, DeprecationWarning,
-                      stacklevel=_stacklevel_above_module(__name__))
+
+        # Ideally, we won't accept keybinding values specified as Python number
```

Comment:
Picky comment but should we use something other than we, i.e. pywbem???

Reply:
DONE. Changed to "pywbem".